14-include.md

```diff
@@ -1,29 +1,39 @@
-## Dependency on other compose projects
+## Include
 
-A Compose application can declare dependency on another compose application, managed by another team
-or shared with others. This allows to keep a compose file reasonably complicated for the limited
-amount of resources a single team has to declare for it's own sub-domain within a larger deployment.
+A Compose application can declare dependency on another Compose application that is managed by another team
+or shared with others. This is useful if:
+- You want to reuse other Compose files
+- You need to factor out parts of your application model into separate Compose files so they can be managed separately or shared with others.
+- Teams need to keep a Compose file reasonably complicated for the limited amount of resources it has to declare for it's own sub-domain, within a larger deployment.
```

Comment:
Thanks for looking at the suggestions @aevesdocker! I still think here, we should have a more clear delineation – A Compose file can declare dependency on another Compose file [full stop]. Whether that other Compose application is managed by another team or shared with others is irrelevant to whether a Compose application [**can**] declare dependency, and should live in the bullet points where it's explained where this can be useful.